pkg/injector/injector.go

```diff
@@ -139,16 +142,16 @@ func getServiceAccount(ctx context.Context, kubeClient kubernetes.Interface, all
 	}
 
 	allowedUids := []string{}
-
+Loop:
 	for _, allowedServiceInfo := range allowedServiceAcccountInfos {
 		serviceAccountInfo := strings.Split(allowedServiceInfo, ":")
 		for _, sa := range serviceaccounts.Items {
 			if sa.Name == serviceAccountInfo[0] && sa.Namespace == serviceAccountInfo[1] {
 				allowedUids = append(allowedUids, string(sa.ObjectMeta.UID))
-				break
+				continue Loop
```

Comment:
If we find uid, break then jump to next, why need a loop here?

Reply:
Can you revert this to break and then on the log.Warnf line skip the log if we found a UID?

Reply:
I reverted to break and corrected a typo of `allowedServiceAcccountInfos`.